src/backend/distributed/sql/citus--10.0-2--10.0-3.sql

```diff
@@ -0,0 +1,5 @@
+-- citus--10.0-2--10.0-3
+
+DROP FUNCTION IF EXISTS pg_catalog.citus_update_table_statistics(regclass);
```

Comment:
when possible, it's a bit nicer to do CREATE OR REPLACE instead of DROP+CREATE. The former preserves things like grants.